geoips/plugins/yaml/product_defaults/tpw/TPW-CIMSS.yaml

```diff
@@ -15,8 +15,15 @@ spec:
         inverse: null
   colormapper:
     plugin:
-      name: tpw_cimss
-      arguments: {}
+      name: matplotlib_linear_norm
```

Comment:
@evrose54 Do you think it is worth deleting these product_defaults altogether, and just fully specifying them in the products/mimic.yaml ?

Reply:
No I don't. I think it makes mimic.yaml look much cleaner, and on the off chance we end up using these product_defaults again for other products, it'd be nice to just reference these yamls instead of always having to fully specify them in a product.

Reply:
Sounds fair